src/components/number-picker/number-picker.component.ts

```diff
@@ -43,6 +43,9 @@ export class NumberPickerComponent implements ControlValueAccessor, OnDestroy, O
     /** Define the precision of floating point values */
     @Input() precision: number = Number.MAX_SAFE_INTEGER.toString().length - 1;
 
+    /** The placeholder text which appears in the text input area when it is empty. Sasi*/
+    @Input() placeholder: string = '';
```

Comment:
I suggest leaving this undefined by default, which should result in no `placeholder` attribute being rendered. This matches our current behavior. The input `placeholder` attribute should only appear when a value has been provided for the new `placeholder` input.

Reply:
Done the changes